content/en/apps/guides/integrations/openmrs.md

```diff
@@ -134,7 +134,7 @@ Remember to convert and upload your forms
 medic-conf --url=https://<username>:<password>@localhost --accept-self-signed-certs convert-contact-forms upload-contact-forms convert-app-forms upload-app-forms
 ```
 
-Remember to setup the [outbound push]({{< ref "apps/reference/app-settings/outbound" >}}) modules to send data to OpenMRS.
+**Note**: Remember to setup the [Outbound push]({{< ref "apps/reference/app-settings/outbound" >}}) modules to send data to OpenMRS.
```

Comment:
```suggestion
{{% alert title="Note" %}}Remember to setup the [Outbound push]({{< ref "apps/reference/app-settings/outbound" >}}) modules to send data to OpenMRS. {{% /alert %}}
```
Updated this to [blockquote styling](https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/#notes-and-tips).